tools/get-images-1.7-stable.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+#
+# This script returns list of container images that are managed by this charm and/or its workload
+#
+# static list
+STATIC_IMAGE_LIST=(
+)
+# dynamic list
+git checkout origin/track/1.7
```

Comment:
this this is a copy/paste error
```suggestion
git checkout origin/track/2.0
```